content-repo/extra-docs/packs/CaptureTheFlag.md

```diff
@@ -0,0 +1,96 @@
+---
+id: capture-the-flag-preparation
+title: Prepare your instance for Capture The Flag
+description: How to prepare your XSOAR instance to run XSOAR's Capture The Flag challenge.
+---
+![ xsoar ctf csi image](../../../docs/doc_imgs/reference/CaptureTheFlag/ctfcsiimage.jpg)
+
+# Introduction
+The XSOAR 8 CTFs provide a fun and engaging way for your audience or teams to learn about Cortex XSOAR.
+The CTFs can be completed in 1 hour and so can be easily incorporated into an event for SOC practitioners. 
+
+
+## Install CTF content Pack
+
+The XSOAR CTF consists of two Content Packs. 
+It is recommended to install "Capture The Flag - 01" and "Capture The Flag - 02" in this order. 
+Navigate to the Cortex Marketplace, search for "Capture The Flag - 01" and click install. After "Capture The Flag - 01" is installed, search for "Capture The Flag - 02" and click Install. Once the CTFs are installed you will see a green check mark against the CTF packs. 
+ 
+
+![ xsoar ctf csi image](../../../docs/doc_imgs/reference/CaptureTheFlag/marketplace_search_for_ctf.png)
+
+# Prepare Your CTF playbook.
+After you install the CTF 1 and CTF 2 content packs from the Marketplace. 
+You must run the “Prepare your Playbook” wizard that will instruct you to configure instances that are required for 
+running the CTF and it will also check the system for any missing items.
+
+![ xsoar ctf csi image](../../../docs/doc_imgs/reference/CaptureTheFlag/marketplace_search_for_ctf.png)
+
+
+## Install and Configure VirusTotal and Unit42 Atom Feed
+The Unit 42 v2 feed provides access to published IOCs that contain known malicious indicators. 
+VirusTotal will analyze suspicious hashes, URLs, domains, and IP addresses. You will need these two integrations for the CTF.
```

Comment:
The VirusTotal content pack has 4 integrations in it. Which integration does the user need?